src/wagtail_personalisation/templatetags/wagtail_personalisation_tags.py

```diff
@@ -0,0 +1,60 @@
+from django import template
+from django.template import TemplateSyntaxError
+
+from django.utils.safestring import mark_safe
+
+from wagtail_personalisation.models import Segment
+from wagtail_personalisation.utils import parse_tag
+
+register = template.Library()
+
+
+def do_segment(parser, token):
+    """Block that only shows content if user is in chosen segment.
+
+    """
```

Comment:
Should be on the end of the first line:
``"""Block that only shows content if user is in chosen segment."""`